package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "@coinbase-sample/prime-sdk-ts",
-  "version": "0.2.1",
-  "main": "dist/main.js",
+  "version": "0.3.0",
```

Comment:
Did we break any existing endpoints? Was there a significant change in the underlying http library/call logic?If not, should this be 0.2.2?

Reply:
one endpoints moved from the portfolio service to financing service. there were 2 added required fields to the wallet endpoints. 0.3 is semver appropriate